src/tests/unit/send-get/CMakeLists.txt

```diff
@@ -3,11 +3,35 @@ caf_compile_executable(get_array get_array_test.f90)
 #caf_compile_executable(get old_get_array_test.f90)
 #target_link_libraries(is_this_still_needed OpenCoarrays)
 
+## Pure get() tests
 caf_compile_executable(get_self sameloc.f90)
-caf_compile_executable(send_array send_array_test.f90)
+caf_compile_executable(get_convert_nums get_convert_nums.f90)
+caf_compile_executable(get_convert_char_array get_convert_char_array.f90)
 caf_compile_executable(get_with_offset_1d get_with_offset_1d.f90)
 caf_compile_executable(whole_get_array whole_get_array.f90)
 caf_compile_executable(strided_get strided_get.f90)
+caf_compile_executable(get_with_vector_index get_with_vector_index.f90)
+
+## Pure send() tests
+caf_compile_executable(send_array send_array_test.f90)
+caf_compile_executable(send_convert_char_array send_convert_char_array.f90)
+caf_compile_executable(send_with_vector_index send_with_vector_index.f90)
+
+# Pure sendget() tests
+caf_compile_executable(sendget_convert_char_array sendget_convert_char_array.f90)
+#caf_compile_executable(sendget_with_vector_index sendget_with_vector_index.f90)
```

Comment:
@vehre Is there an additional test for this that wasn't committed? (sendget_with_vector_index.f90)

Reply:
Nope, none that I can find. Presumably I just planned to do it, but never did and the commented cmake-line just remained there out of laziness.